### ISSUE TITLE
Can not get correct results

### ISSUE THREAD

**Issue by SeaMountains** [LYlala/RCF-tensorflow#2]
maybe the loss function is wrong
